lib/app/migrate.ts

```diff
@@ -179,21 +179,30 @@ async function selectAppToMigrate(
   if (migratableComponents.length !== 0) {
     logger.log(
       lib.migrate.componentsToBeMigrated(
-        `\n - ${migratableComponents.join('\n - ')}`
+        `\n - ${[...new Set(migratableComponents)].join('\n - ')}`
```

Comment:
These need to be deduped before we log them

Reply:
Do dupes exist when the user has multiple instances of a given component? Like multiple cards?

Reply:
Yeah, they can have multiple cards, webhooks, etc.